File: YukariLauncher/src/main/java/net/kdt/pojavlaunch/progresskeeper/TaskCountListener.java
package net.kdt.pojavlaunch.progresskeeper;

/**
 * タスク数の変更をリッスンするインターフェース。
 */
public interface TaskCountListener {
    /**
     * タスク数が更新されたときに呼び出されます。
     * @param taskCount 現在のタスク数
     */
    void onUpdateTaskCount(int taskCount);
}

File: YukariLauncher/src/main/java/com/arata/yukarilauncher/task/TaskCountListener.kt
package com.arata.yukarilauncher.task

/** タスク数の変更を監視するリスナー。 */
fun interface TaskCountListener {
    /** タスク数が変更されたときに呼び出される。 @param taskCount 現在のタスク数 */
    fun onUpdateTaskCount(taskCount: Int)
}
